src/components/MainSidebar/YourDimensionsPanel/YourDimensionsPanel.js

```diff
@@ -0,0 +1,49 @@
+import i18n from '@dhis2/d2-i18n'
+import { Input } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { useDebounce } from '../../../modules/utils.js'
+import { DimensionsList } from '../DimensionsList/index.js'
+import { useYourDimensions } from './useYourDimensions.js'
+import styles from './YourDimensionsPanel.module.css'
+
+const YourDimensionsPanel = ({ visible }) => {
+    const [searchTerm, setSearchTerm] = useState('')
+    const debouncedSearchTerm = useDebounce(searchTerm, 500)
```

Comment:
Now that `useDebounce` is used in multiple places, maybe the default debounce time should be exported as a separate prop? (e.g. `useDebounce(searchTerm, DEFAULT_DEBOUNCE_TIME)`) or let `useDebounce` set the default time internally and only pass `searchTerm` without an explicit time (e.g. `useDebounce(searchTerm)`)?

Reply:
🤔 I think we might need to export two values, one for when a user is typing, and the other for when we want to respond to events that fire often (i.e. `mousemove`, `resize`, etc). Maybe like this:
```js
export const DEFAULT_USER_INPUT_DELAY = 500
export const DEFAULT_WINDOW_EVENT_DELAY = 150
```
FYI: I already used the shorter time yesterday for setting the table max-width after a window resize.

Reply:
Sounds good to me 👍

Reply:
I just saw one being `150` and figured we could have a `default` and a custom. But two "defaults" works as well

Reply:
> or let `useDebounce` set the default delay internally and only pass `searchTerm` without an explicit time (e.g. `useDebounce(searchTerm)`)?

Actually this was a very nice idea. I've gone with that.